infra/terraform/modules/data_access/api_gateway.tf

```diff
@@ -0,0 +1,77 @@
+resource "aws_api_gateway_rest_api" "analytics" {
+  name = "${var.env}_analytics"
+}
+
+# API Gateway stage (to give access to API from the outside)
+resource "aws_api_gateway_deployment" "analytics" {
+  rest_api_id = "${aws_api_gateway_rest_api.analytics.id}"
+  stage_name  = "${var.env}"
```

Comment:
As the stage becomes the last part of the API URL (e.g. `https://rand_stuff.execute-api.eu-west-1.amazonaws.com/dev/github`) it's really clear which environment will end up handling the webhook which to me is a benefit as it avoid surprises/confusion.

But, it's easy to change and I can if you really think it's better to have something like `https://rand_stuff.execute-api.eu-west-1.amazonaws.com/api/github` :)